### DISCUSSION TITLE
Error thrown on SequelizeStorage construction

### DISCUSSION THREAD

**Discussion by mrghosti3** [sequelize/umzug#557] [Q&A] [answered]
Hello, I get an error while calling `SequelizeStorage` constructor:

```
Error: One of "sequelize" or "model" storage option is required
    at new SequelizeStorage (/ddisk/projects/testmig/node_modules/umzug/lib/storage/sequelize.js:15:19)
```

I am creating it using just like the example in [master](https://github.com/sequelize/umzug#minimal-example) branch shows:
```js
//my snippet
const storage = new SequelizeStorage({ sq });
```
I tried using format `new SequelizeStorage(sq)` but I would run into different error pertaining `_b` variable. So my question what is the right way to init `SequelizeStorage`?

`package.json`:
```json
{
  "type": "module",
  "dependencies": {
    "mariadb": "^3.0.0",
    "sequelize": "^6.20.1"
  },
  "devDependencies": {
    "dotenv": "^16.0.1",
    "umzug": "^3.1.1"
  }
}
```

**Comment by mmkal** [comment, marked as answer]
> Hello, I get an error while calling `SequelizeStorage` constructor:
> 
> ```
> Error: One of "sequelize" or "model" storage option is required
>     at new SequelizeStorage (/ddisk/projects/testmig/node_modules/umzug/lib/storage/sequelize.js:15:19)
> ```
> 
> I am creating it using just like the example in [master](https://github.com/sequelize/umzug#minimal-example) branch shows:
> 
> ```js
> //my snippet
> const storage = new SequelizeStorage({ sq });
> ```

Maybe I’m missing something, but it seems you aren’t creating it like the example. You’re passing `{ sq }` to the constructor and the example passes `{ sequelize }` - that’s what the error message is saying too. If your sequelize instance is named `sq`, you could use `new SequelizeStorage({ sequelize: sq })`.

**Reply by mrghosti3** [reply]
Yeah, my bad. I though that all I needed is to give it `Sequelize` instance and naming doesn't matter. Thank you very much for the help.
